src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const styleCommands = sourceString.split(';');
```

Comment:
According to the [CODE STYLE] checklist item, avoid variable namings like `array`, `object`, `arr`, `obj`, `result`. The variable name `styleCommands` could be more descriptive, like `styleDeclarations` or `cssDeclarations`, to better reflect its contents.